modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -494,10 +498,13 @@
 			// Return a "textSource" object:
 			return new mw.TextSource( embedSource, _this.embedPlayer );
 		},
-		forceLoadLanguage: function(){
-			var lang = this.getConfig('forceLoadLanguage');
+		forceLoadLanguage: function(language){
+			var lang = this.getConfig('forceLoadLanguage') ? this.getConfig('forceLoadLanguage') : language;
```

Comment:
@OrenMe  
Because I want this function to also serve the new API from line 156 so that external plugins will be able to load a specific language. 
Example - external plugin gets "fr" from metadata and is interested to load French, it will send a notification "forceLoadCaptions" with "fr" as argument.